app/src/androidTest/java/ch/epfl/swissteam/services/FirebaseTest.java

```diff
@@ -0,0 +1,37 @@
+package ch.epfl.swissteam.services;
+
+import android.support.test.runner.AndroidJUnit4;
+
+import com.google.firebase.database.FirebaseDatabase;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+
+/**
+ * Class designed to wrap tests that interact with the Firebase Realtime Database to prevent actual interaction with the DB.
+ *
+ * @author Adrian Baudat
+ */
+@RunWith(AndroidJUnit4.class)
+public abstract class FirebaseTest {
+
+    @Before
+    public void setUp() {
```

Comment:
Maybe setUp and tearDown should be final to avoid subclasses to override them
```suggestion
    final void setUp() {
```